docs/tutorials/subscription-platform.md

```diff
@@ -76,13 +76,15 @@ Product Names are the canonical displayed name shown in Sub Plat UI. In some cas
 | product:privacyNoticeURL:{locale}         | Optional. Localized override URL for the webpage containing the Privacy Notice for the product offering. |
 | product:privacyNoticeDownloadURL          | Optional. The URL for a downloadable version of the Privacy Notice for the product offering. This has the same requirements as product:termsOfServiceDownloadURL. |
 | product:termsOfServiceURL:{locale}        | Optional. Localized override URL for the webpage containing the Terms of Service for the product offering. |
+| product:cancellationSurveyURL             | Optional. Override URL for the Cancellation Survey for the product offering. |
```

Comment:
should we add the default url?

Reply:
Can you put this in alphabetical order (and maybe while you're at it do the same for the promotionCodes key among the optional keys list)?

I wouldn't mention the default URL until we have one that is generic. Currently we've hardcoded in the P&S products cancellation survey into the code. I filed https://mozilla-hub.atlassian.net/browse/FXA-4712 for us to move that URL to this new metadata key and make a call on whether we should have a default URL in the first place.

I don't see this precedent elsewhere, but it might be nice to mention where this cancellation URL shows up (i.e. in the email when the user cancels in this way).

Finally, once this is merged, can you let folks know in the #subscription-platform channel and maybe add an agenda item for the next NA/Euro SubPlat office hours? I think RPs will be happy to hear they can have a custom survey here now.